src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -759,7 +759,14 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                 $newline = [Environment]::Newline
                                 $output = [System.Text.StringBuilder]::new()
                                 $prefix = ' ' * $indent
-                                $accentColor = Get-VT100Color $Host.PrivateData.FormatAccentColor
+
+                                if ($null -ne $Host.PrivateData.FormatAccentColor) {
+                                    $accentColor = Get-VT100Color $Host.PrivateData.FormatAccentColor
+                                }
+                                else {
+                                    $accentColor = Get-VT100Color [Console]::ForegroundColor
+                                }
```

Comment:
perfect time to use the ternary op?

Reply:
null coalescing is even better here

Reply:
I tried a version with both ternary and null coalescing.  Works but hard to read.  Going to not use ternary for this case.

Reply:
Still on that future syntax 😎